docs/Modpacks/Other-Topics/Adding-and-Removing-Recipes.md

```diff
@@ -110,7 +133,9 @@ ServerEvents.recipes(event => {
 })
 ```
 
-1. Note due to a `JS Moment` you have to force `scannerResearch` to be interpreted by Rhino in a specific way. Scanner Research accepts an `ItemStack` input in the `.researchStack()` object, and you can also define the `EUt` and `Duration` outside of the `.researchStack()` object.
+1. Note due to a `JS Moment` you have to force `scannerResearch` to be interpreted by Rhino in a specific way. 
```

Comment:
pls make JS Moment not be a code block it's so bad

Reply:
I'm just gonna reword it entirely